lib/metric_fu/metrics/reek/generator.rb

```diff
@@ -8,21 +10,18 @@ def emit
       files = files_to_analyze
       if files.empty?
         mf_log "Skipping Reek, no files found to analyze"
-        @output = run!([], config_files)
+        run!([], config_files)
```

Comment:
I tried creating specs by making yaml fixtures storing the content of @output from the different major Reek versions (1,2,3,4). 

But I fail deserialising them because I need the proper Reek classes for that version. I think I might need to serialise it to basic data types..but then I should implement the parser to work on this (meaning that we need to convert it to basic data on normal run as well)

So any suggestions on how I can verify the parsing of the Reek data structure?

ps: will push my work in progress specs as pending